.github/osmoql/src/check-sdk-operators.ql

```diff
@@ -0,0 +1,26 @@
+/**
+ * @name Wrong operator between sdk types.
+ * @kind problem
+ * @problem.severity warning
+ * @id osmo-ql/wrong-binary-operators
+ */
+
+/* 
+problem.severity is set to warning, because in some cases using native operators between sdk.Int(s) or sdk.Dec(s) is justified
+ex: https://github.com/osmosis-labs/osmosis/blob/main/x/twap/logic.go#L58-L63
+*/
+
+import go
+
+from BinaryExpr b
+where
+  (b.getOperator() = "==" or b.getOperator() = "<" or b.getOperator() = ">" or b.getOperator() = "<=" or b.getOperator() = ">=")
```

Comment:
note: actually, we only need to check for `==` operator, because other operators are not defined on go structs, which will raise an error in the editor either ways

Reply:
Lets just check equals then?